src/main.js

```diff
@@ -20,10 +20,6 @@ Vue.use(Toast, {
   timeout: 2000
 })
 
-import PerfectScrollbar from 'vue2-perfect-scrollbar'
```

Comment:
You forgot to remove it in both SeachModal.vue & TorrentDetailModal > Peers.vue
It can also be removed as a dependency from package.json :)